app/assets/javascripts/comments.js

```diff
@@ -83,7 +85,7 @@ $(() => {
       <span class="has-float-right has-font-size-caption js-character-count-comment-body"
             data-max="1000" data-min="15">
         <i class="fas fa-ellipsis-h js-character-count__icon"></i>
-        <span class="js-character-count__count">0 / 1000</span>
+        <span class="js-character-count__count">`+length+` / 1000</span>
```

Comment:
In a string wrapped with <code>`</code> backticks, you can use string interpolation, like Ruby:

```suggestion
        <span class="js-character-count__count">${content.length} / 1000</span>
```